src/components/AddComment.jsx

```diff
@@ -0,0 +1,57 @@
+import { useState } from "react";
+import { postComment } from "../api";
+
+const AddComment = (props) => {
+	const [userComment, setUserComment] = useState({
+		username: "cooljmessy",
+		body: "",
+	});
+
+	const handleSubmit = (event) => {
+		if (userComment.body === "") {
+			event.preventDefault();
```

Comment:
do we need to an event default in both if/else? could we just put it before the if?

Reply:
shouldn't be changing the DOM. conditionally render using an error state